packages/api/tests/post-moderation.test.ts

```diff
@@ -0,0 +1,46 @@
+import { moderatePost } from '../src'
+import type {
+  ModerationBehaviors,
+  ModerationBehaviorScenario,
+} from '../definitions/moderation-behaviors'
+import { ModerationBehaviorSuiteRunner } from './util/moderation-behavior'
+import { readFileSync } from 'fs'
+import { join } from 'path'
+
+const suite: ModerationBehaviors = JSON.parse(
+  readFileSync(
+    join(__dirname, '..', 'definitions', 'post-moderation-behaviors.json'),
+    'utf8',
+  ),
+)
+
+const suiteRunner = new ModerationBehaviorSuiteRunner(suite)
+
+describe('Post moderation behaviors', () => {
+  const scenarios = Array.from(Object.entries(suite.scenarios))
+  it.each(scenarios)(
+    '%s',
+    (_name: string, scenario: ModerationBehaviorScenario) => {
+      const res = moderatePost(
+        suiteRunner.postScenario(scenario),
+        suiteRunner.moderationOpts(scenario),
+      )
+      expect(res.content).toBeModerationResult(
```

Comment:
The types for the extended `expect` matcher are not making it through here for me, so `toBeModerationResult()` appears not to exist.  I did a brief search to see if it's easy to extend the types on jest's `expect`, but nothing immediately jumped out at me.  I don't think it needs to block this work at all, but would be nice for us to eventually resolve so that the types in the test suite all validate.